MCB-Nucleo/Core/Inc/MCB.h

```diff
@@ -2,45 +2,49 @@
  * MCB.h
  *
  *  Created on: Jun. 1, 2023
- *      Author: kcgro
+ *      Author: Kyle Groulx
  */
 
 #ifndef INC_MCB_H_
 #define INC_MCB_H_
 
 #include "can.h"
 
-#define CAN_DATA_LENGTH 8
-#define ADC_POLL_DELAY 10
-#define SEND_MOTOR_COMMAND_DELAY 10
-#define ADC_DEADZONE 500
-#define ADC_MAX 4096
+#define CAN_DATA_LENGTH 8			// Length of a CAN message in bytes
+
+#define UPDATE_FLAGS_DELAY 5		// updateFlags rtos task delay time in ms
```

Comment:
Just a note. You can also do UPDATE_FLAGS_TASK_DELAY_MS, just a style option which would make it longer but would remove the need to inspect the macro to read the comment about the macro. This is good as is.